webapp/i18n/index.ts

```diff
@@ -0,0 +1,13 @@
+import * as en from './en.json';
+import * as es from './es.json';
+
+export function getTranslations(locale: string): {[key: string]: string} {
+    switch (locale) {
+    case 'en':
+        return en;
+    case 'es':
+        return es;
+    }
+    return {};
+}
```

Comment:
Out of scope: I would love a generic function where no additional locals have to get added.

Reply:
what do you mean with additional locals?

Reply:
When a contributor adds a new language like `de`, this function has to get changed.

Reply:
Yes, we can try to change it later, but I think that is not a big deal, we are not going to have hundreds of new translations, so... the work to do in this cases is minimal, probably automatize that part is going to be more expensive that just doing it manually when we add a new language.

Reply:
Given that this code will be (hopefully) used by all webapp plugin, there are a lot of changes needed in the future.

But surely my comments are not blocking.